03/app.js

```diff
@@ -41,18 +41,32 @@ books.getAuthor = function(isbn) {
     return false;
 }
 
-books.getTitle = function(isbn, lang) {
+books.getTitle = function (isbn, lang) {
+    if (typeof this[isbn]["title"] === "undefined" && typeof this[isbn]["title"] === null) {
+        return null;
+    }
```

Comment:
Ten warunek nigdy nie będzie prawdą: `typeof this[isbn]["title"] === null` - `typeof` zawsze zwraca ciąg znaków (string). Jeśli sprawdzamy `null` to tak: `this[isbn]["title"] === null`